src/lang/translator.cpp

```diff
@@ -34,6 +34,9 @@ bool Translations::RegisterProvider(uint16_t langCode, const ITranslationProvide
 }
 
 bool Translations::ChangeLanguage(uint16_t langCode) {
+    if (langCode == 0) {
+        return false; // requesting an invalid langCode to change to
+    }
```

Comment:
/question/ Why is this case handled here explicitly? If zero is not a valid code, no language should have it assigned and we should return `false` from line 46. Or am I missing something?